### ISSUE TITLE
local invoke with --no-build flag(s) causes deprecation warnings

### ISSUE THREAD

**Issue by thehideki81** [serverless-heaven/serverless-webpack#965]
# This is a (Bug Report)

## Description

Using --no-build (and --webpack-offline) causes deprecation warnings

`serverless invoke local --no-build --webpack-offline -f ${FUNCTION}`

```
Detected unrecognized CLI options: "--build", "--webpack-offline".
Starting with the next major, Serverless Framework will report them with a thrown error
More Info: https://www.serverless.com/framework/docs/deprecations/#UNSUPPORTED_CLI_OPTIONS
```

## Additional Data

    "serverless": "^2.44.0",
    "serverless-offline": "8.2.0",
    "serverless-webpack": "^5.5.0",
    "webpack": "^5.4.0"

**Comment by thehideki81** [comment]
I'm currently using a custom plugin to circumvent this issue, can be used as a helper if this issue is not fixed before Serverless Offline 3.0
```
'use strict'

class ServerlessPlugin {
  constructor(serverless, options) {
    this.hookEnabled = false
    this.serverless = serverless
    this.options = options
    this.individually = this.serverless.service.package.individually

    this.hooks = {
      'before:invoke:local:invoke': this.prepareWebpackOfflineInvoke.bind(this),
      'after:invoke:local:invoke': this.finishWebpackOfflineInvoke.bind(this)
    }
  }

  prepareWebpackOfflineInvoke() {
    // Use normal services folder for invokes
    if (this.individually && process.env.IS_OFFLINE) {
      this.hookEnabled = true
      // Serverless from 3.x doesn't support custom CLI parameters.
      // We do a trick to provide some options to serverless-webpack through this plugin.
      // Revisit this when this maybe fixed:
      // https://github.com/serverless-heaven/serverless-webpack/issues/965
      this.serverless.processedInput.options.build = false
      this.serverless.processedInput.options.webpack_offline = true
      // Disable individual packaging. Fixes serverless-webpack usage without building again.
      this.serverless.service.package.individually = false
    }
  }

  finishWebpackOfflineInvoke() {
    // change back
    if (this.hookEnabled) {
      this.serverless.service.package.individually = true
    }
  }
}

module.exports = ServerlessPlugin

```

**Comment by vicary** [comment]
CLI options are more restrictive now, though it still works. One of the options I could think of is falling back to serverless.yml, e.g. `custom.webpack.build` so users can use the `${file:(foo.js)}` syntax without making a custom plugin.
